### PR TITLE
minor fix in e2e test

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/testadapters.yaml
Comment (leecalcote) [inline .github/workflows/testadapters.yaml:111]: Ugh. More sleeping.

@hexxdump have you arrived at a solution for `mesheryctl system start`'s sleeping? // @piyushsingariya

File: .github/workflows/testadapters.yaml
Comment (leecalcote) [inline .github/workflows/testadapters.yaml:113]: I have a hard time understanding why `mesheryctl system status` isn't used here. Why isn't it? If something needs changed, are have those changes been requested? // @meshery/mesheryctl-maintainers

File: .github/workflows/testadapters.yaml
Comment (Revolyssup) [reply .github/workflows/testadapters.yaml:111]: I am not sleeping here. We are polling. This time is just the timeout, untill when to POLL

File: .github/workflows/testadapters.yaml
Comment (Revolyssup) [reply .github/workflows/testadapters.yaml:113]: To get the pod logs, I need the name of the pods. mesheryctl doesn't quite give the name of the pods

Comment (leecalcote) [reply .github/workflows/testadapters.yaml:113]: Please file an enhancement request for a `-v` verbose flag for this command.

File: .github/workflows/testadapters.yaml
Comment (leecalcote) [reply .github/workflows/testadapters.yaml:111]: Ok. So, this is sleeping with a chance for early exit. ;)

How can we do this better? @hexxdump?

Comment (Revolyssup) [reply .github/workflows/testadapters.yaml:111]: No, not sleeping, we keep running a specific command and keep checking if the output is as desired. When we get the desired output, we exit. This time is just the max timeout after which we dont further run that specific command

File: .github/workflows/testadapters.yaml
Comment (leecalcote) [reply .github/workflows/testadapters.yaml:113]: Good to merge after this issue is filed.

**Review by leecalcote — APPROVED** [review]

File: .github/workflows/testadapters.yaml
Comment (leecalcote) [reply .github/workflows/testadapters.yaml:111]: Related - https://github.com/meshery/meshery/issues/4726

Comment (leecalcote) [reply .github/workflows/testadapters.yaml:111]: My sarcasm wasn't thick enough. I'm saying that this polling approach is marginally better than sleeping and in its basic essence, the same thing.

File: .github/workflows/testadapters.yaml
Comment (Revolyssup) [reply .github/workflows/testadapters.yaml:113]: https://github.com/meshery/meshery/pull/4845 merge this

Comment (piyushsingariya) [reply .github/workflows/testadapters.yaml:113]: You don't need the pod-name to get logs, just the component-name is enough.
```
❯ mesheryctl system logs meshery-app-mesh
Starting Meshery logging...
meshery-app-mesh-d8bb6fb98-4qflj        |       Could not load definitions and schemas for static component registeration: lstat /templates/oam/workloads: no such file or directorytime="2022-01-02T08:38:36Z" level=info msg="Adaptor Listening at port: 10005" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:36Z" level=info msg="Starting static component registration..." app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:36Z" level=info msg="Static Component registration completed" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:38:37Z" level=info msg="Registering latest workload components for version v1.4.2" app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |       Looking for kubeopenapi-jsonschema in  /root/.meshery/bin
meshery-app-mesh-d8bb6fb98-4qflj        |       Downloading kubeopenapi-jsonschema at /root/.meshery/bin/kubeopenapi-jsonschema...
meshery-app-mesh-d8bb6fb98-4qflj        |       Download Completed
meshery-app-mesh-d8bb6fb98-4qflj        |       time="2022-01-02T08:39:06Z" level=info msg="Latest workload components successfully registered." app=app-mesh-adapter
meshery-app-mesh-d8bb6fb98-4qflj        |
```

**Pull request merged**
